.github/workflows/ci.yml

```diff
@@ -58,13 +58,13 @@ jobs:
     timeout-minutes: 5
     env:
       # renovate: datasource=github-releases depName=rhysd/actionlint
-      ACTIONLINT_VERSION: '1.7.4'
+      ACTIONLINT_VERSION: "1.7.4"
     steps:
-      - uses: actions/checkout@v4
-      - run: echo "::add-matcher::.github/actionlint-matcher.json"
-      - run: |
-          bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/main/scripts/download-actionlint.bash) $ACTIONLINT_VERSION
-          ./actionlint -shellcheck '' -ignore 'property "vault_.+" is not defined in object type' -ignore 'object type "{}" cannot be filtered by object filtering `.*` since it has no object element'
+      - uses: camunda/infra-global-github-actions/actionlint@main
```

Comment:
For demo purposes you might want to use `@gh-620-centralized-infracheck-workflow` here since your changes are not on `main` branch of `camunda/infra-global-github-actions` yet - thus failing CI.

Reply:
The CI will most likely also fail because the `-ignore` params are not preserved in this change, even though [your action supports it](https://github.com/camunda/infra-global-github-actions/blob/gh-620-centralized-infracheck-workflow/actionlint/action.yml#L10).